boards/ct/ctcc/ctcc_nrf52840.yaml

```diff
@@ -9,6 +9,8 @@ toolchain:
 supported:
   - ble
   - gpio
+  - usb_cdc
```

Comment:
Why?

Reply:
It seems this doesn't exist anymore. Removed, sorry!

Reply:
It was actually never used and I cleaned it up twice. I am curious why you added it, maybe I missed something?

Reply:
It was in `v3.1.0` Zephyr AFAIR in `nrf52840dongle_nrf52840.yaml`. I worked for some time on old `v3.1.0` and I added it here but I forgot to check if this entry is still used